### ISSUE TITLE
invoke server method,how get return value,eg string

### ISSUE THREAD

**Issue by MakerXia** [adamhartford/SwiftR#11]
invoke server method,how get return value,eg string,thank you
public string SendComplex(ComplexMessage message)  {
        
      Clients.All.notifyComplex (message);
      return "OK"
 }

**Comment by adamhartford** [comment]
Hi, I'll need some more information to help. Can you post some of your client code? In general, you should have something like this:

```swift
SwiftR.connect("http://yourserver.com") { connection in
    self?.complexHub = connection.createHubProxy("complexHub")

    self?.complexHub.on("notifyComplex") { args in
        let m: AnyObject = args!["0"] as AnyObject!
        print(m)
    }
}
```

**Comment by MakerXia** [comment]
I need some method like SignalR-objc method,like below code
renderHub?.invoke("hello", withArgs: renderArgs, completionHandler: { (response, error) -> Void in         
      print("\(response)")    
 })
thank you

**Comment by adamhartford** [comment]
OK, I see. I haven't added support for `invoke` return values yet. I agree that this would be nice to have. I'll work on it next.

**Comment by adamhartford** [comment]
Commit fb4fdce4c9a2a9ae95be2a4f63f2c46beae953dd adds this support. The invoke signature is now:

```swift
public func invoke(method: String, arguments: [AnyObject]?, callback: ((AnyObject?, String?) -> ())? = nil)
```

Here's an example usage:

```swift
simpleHub.invoke("sendSimple", arguments: ["Simple Test", "This is a simple message"]) { (result, error) in
    if let e = error {
        print("Error message: \(e)")
    } else {
        print("Success!")
        if let r = result {
            print("Result: \(r)")
        }
    }
}
```

**Comment by MakerXia** [comment]
Thank you!

**Comment by adamhartford** [comment]
Change is included in v0.8.1.
